add_vector_to_row_5.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def add_vector_to_row_5(vec, matrix):
+    """
+    Create new vector after add each row of the matrix by the given vector.
+    :param vec:(list) Given vector to add to each row in the matrix.
+    :param matrix:(np array) Given matrix.
+    :return: New vector: the result of add the given vector to each row.
+    """
+    result = np.empty_like(matrix)
+    for i in range(len(matrix)):
+        result[i, :] = matrix[i, :] + vec
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html